### DISCUSSION TITLE
Suppression de compte

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1416] [Idées]
Il me semble qu'on a pas mal de requêtes pour la suppression de compte - est-ce qu'on veut ajouté l'option pour l'utilisateur de le faire soi-même pour réduire un peu les responsabilités des bizdevs ?

**Comment by alemangui** [comment]
À voir si ce n'est un pic suite aux différentes campagnes mailing de Gamma.
